### DISCUSSION TITLE
2023/07/17/Hexo%E8%87%AA%E5%8A%A8%E9%83%A8%E7%BD%B2/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#15] [Announcements]
# 2023/07/17/Hexo%E8%87%AA%E5%8A%A8%E9%83%A8%E7%BD%B2/

It's Florance!

https://florance.eu.org/2023/07/17/Hexo%E8%87%AA%E5%8A%A8%E9%83%A8%E7%BD%B2/

**Comment by ERA-YES** [comment]
> 教教我教教我😶
